ros_control_boilerplate: 0.1.0-1 in 'indigo/distribution.yaml' [bloom]

davetcoleman opened the pull request:
Increasing version of package(s) in repository `PACKAGE` to `0.1.0-1`:

- upstream repository: https://github.com/davetcoleman/ros_control_boilerplate.git
- release repository: https://github.com/davetcoleman/ros_control_boilerplate-release.git
- distro file: `indigo/distribution.yaml`
- bloom version: `0.5.20`
- previous version for package: `null`

## ros_control_boilerplate

```
* Initial release of ros_control_boilerplate
* Contributors: Dave Coleman
```

davetcoleman commented:
Is this a bug? I ran bloom release again this time with the "Repository Name:" specified. Still it calls my repo "PACKAGE"

davetcoleman commented:
Ok, I manually fixed the commit. I think bloom-release might have a bug where it always says "PACKAGE"

DaikiMaekawa commented:
Are you sure? You using 0.5.20 version, right? Actually, it's fine to me like the link below...

https://github.com/ros/rosdistro/pull/9881

wjwwood commented:
Yeah, I haven't seen anyone else run into this as a bug.

davetcoleman commented:
Ok. I even tried deleting the whole ``-release`` repo and doing it from scratch. But its likely I had an old version of bloom release, I only update when I see a need. I've just updated my Ubuntu. I'll see if I run into the same problem again for a different repo. Thanks.

This is ready to merge because I manually edited and rebased the commit.

wjwwood commented:
The CI is still failing. I think it would be better for you to rerun bloom after updating.

wjwwood commented:
Also, you are using `0.5.20` (it says it in the release repository and in this pr's description), so you're using the correct version.

davetcoleman commented:
Ok, I'll re-bloom.

Also, this package requires https://github.com/ros/cmake_modules/pull/38, will it still be able to build even though you haven't released cmake_modules yet? I assume not now that I think about it.

wjwwood commented:
It will not. Also I wasn't planning on releasing that change into Indigo. It's currently only on the 0.4-devel branch which is Jade only right now.
